src/components/Speciality/HowWeWorkWith.js

```diff
@@ -20,35 +23,73 @@ const StyledRow = styled(Row)`
     padding-top: ${({ theme }) => theme.space[7]};
     padding-bottom: ${({ theme }) => theme.space[7]};
   `}
+
+  ${is('noPaddingBottom')`
+    padding-bottom: 0 !important;  
+    `}
+
+  ${is('noPaddingTop')`
+    padding-top: 0 !important; 
+    `}
+`
+
+const PractiseCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[5]};
+
+  ${breakpoint('tablet')`
+    padding-top: ${({ theme }) => theme.space[6]};
+  `}
 `
 
 const HowWeWorkWithSection = ({
-  howWeWorkWithTitle,
-  howWeWorkWithCopy,
-  howWeWorkWithPractises
+  howWeWorkWithTitle: title,
+  howWeWorkWithCopy: copy,
+  howWeWorkWithPractises: practises = []
 }) => {
+  const hasPractises = practises && practises.length
```

Comment:
I would move the `practises.length > 0` in this logic so you can use `hasPractises &&` below. It's nicer.

Reply:
@GBouffard nice spot, have updated